uw-research-computing/testing-and-scaling-up.md

```diff
@@ -0,0 +1,126 @@
+CPUs (i.e. cores), and the amount of disk and memory that the job needs 
+to complete successfully.
+When you submit a job for the 
+first time you may not know just how much to request and that's OK. 
+Below are some suggestions for making resource requests for initial test 
+jobs. **As always, reviewing the HTCondor `log` file from past jobs is 
+a great way to learn about the resource needs of your jobs.**
+
+**Requesting CPU Cores**
+
+- For **requesting CPU cores start by requesting a single cpu**. With single-cpu jobs, you will see 
+your jobs start sooner. Ultimately you will be able to achieve 
+greater throughtput with single cpus jobs compared to jobs that request 
+and use multiple cpus. 
+
+    - **Keep in mind, requesting more CPU cores for a job 
+    does not mean that your jobs will use more cpus.** Rather, you want to make sure 
+    that your CPU request matches the number of cores (i.e. 'threads' or 'processes') 
+    that you expect your software to use. (Most softwares only use 1 CPU core, by default.)
+
+    - There is limited support for multicore work in our high througput system. For large-scale multicore jobs, contact a Research Computing Facilitator at <chtc@cs.wisc.edu>. 
+    
+**Requesting Disk Space**
+
+- To inform initial disk requests always look at the size of your input 
+files. At a minimumm, you need to request enough disk to support all 
+of the input files, executable, and the output you expect, but don't forget that the standard 'error' and 'output'
+files you specify will capture 'terminal' output that may add up, too.
+      
+    - If many of your input and output files are compressed 
+(i.e. zipped or tarballs) you will need to factor that into your 
+estimates for disk usage as these files will take up additonal space once uncompressed 
+in the job.
+      
+    - For your initial tests it is OK to request more disk than 
+your job may need so that the test completes successfully. **The key 
+is to adjust disk requests for subsequent jobs based on the results 
+of these test jobs.**
+ 
+**Requesting Memory**
+
+- Estimating **memory requests** can sometimes be tricky. If you've performed the 
+same or similar work on another computer, consider using the amount of 
+memory (i.e. RAM) from that computer as a starting point. For instance, 
+most laptop computers these days will have 8 or 16 GB of memory, which is okay to start 
+with if you know a single job will succeed on your laptop.
+
+	- For your initial tests it is OK to request more memory than 
+your job may need so that the test completes successfully. **The key 
+is to adjust memory requests for subsequent jobs based on the results 
+of these test jobs.**
+
+**Importance of Test Jobs**
+
+- Once you have run a test job using a small number of jobs, **Review the bottom of the HTCondor `log` files from your test jobs to see how many cpus and how much memory and disk space were used.** HTCondor will report 
```

Comment:
Can you tie together this section and the following section in some way? Maybe start this section with something like "To fine tune your requests, make sure to run test jobs - see below for a recommended process." and then continue with this paragraph as is.

Reply:
Done!